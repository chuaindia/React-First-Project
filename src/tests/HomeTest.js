import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../components/pages/Home';

it('It will render the Homepage', () => {
  const home = renderer
    .create(<Router><Home /></Router>)
    .toJSON();
  expect(home).toMatchSnapshot();
});
