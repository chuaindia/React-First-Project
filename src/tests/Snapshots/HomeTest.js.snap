exports[`It should render the Homepage 1`] = `
<div>
  <h1
    className="Home_title"
  >
    Math Magicians
  </h1>
  <h2
    className="Home_info"
  >
    Welcome to Our Page
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati .Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.
  </p>
</div>
`;

exports[`Test to render the Home page of the app It should render the Homepage 1`] = `
<div>
  <h1
    className="Home_title"
  >
    Math Magicians
  </h1>
  <h2
    className="Home_info"
  >
    Welcome to Our Page
  </h2>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati .Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam amet consequuntur possimus, nulla id praesentium porro voluptatum laboriosam rerum ullam ipsa. Iusto alias quas, distinctio iste voluptates assumenda exercitationem obcaecati.
  </p>
</div>
`;